add virtual roadshow blog (#247)
AdoptOpenJDK/blog: content/blog/adoptopenjdk-virtual-roadshow/index.md

---
title: AdoptOpenJDK Virtual Roadshow
date: "2020-07-08T12:00:00+00:00"
author: georgeadams
description: Join the AdoptOpenJDK virtual roadshow as we speak at a JUG near you!
featuredImage: "./duke_mask.png"
tags:
  - AdoptOpenJDK
  - Conferences
---
Join the AdoptOpenJDK virtual roadshow as we speak at a JUG near you!

![Duke with mask](./duke_mask.png)

AdoptOpenJDK is doing a virtual roadshow of the Java User Groups. We plan to reach as many groups as possible but we will also live stream the event on YouTube.

## Abstract

> AdoptOpenJDK is rapidly becoming a leading provider of OpenJDK™ binaries. With over 170 million downloads in the last year, it is now a serious contender for your production usage of Java™. AdoptOpenJDK provides prebuilt OpenJDK binaries from a fully open source set of build scripts and infrastructure. This talk will cover how we build on over 15 different platforms, execute over 87 million tests and distribute OpenJDK binaries to millions of users. We will also cover how AdoptOpenJDK binaries compare against the Java binaries that you use today. If you're curious to understand the difference between OpenJDK, Oracle Java, AdoptOpenJDK and all the other distributions, then this is the talk for you!

Reviewer: I think you can be a bit more bold in the opening sentence.  AdoptOpenJDK is *the* leading provider of OpenJDK binaries, and not only a _contender_ for production use, it is ready and in production in many enterprises!

Reviewer 2: ```suggestion
AdoptOpenJDK is the leading provider of OpenJDK™ binaries. With over 170 million downloads in the last year, it is successfully used by many enterprises and ready for your production usage of Java™. AdoptOpenJDK provides prebuilt OpenJDK binaries from a fully open source set of build scripts and infrastructure. This talk will cover how we build on over 15 different platforms, execute over 87 million tests and distribute OpenJDK binaries to millions of users. We will also cover how AdoptOpenJDK binaries compare against the Java binaries that you use today. If you're curious to understand the difference between OpenJDK, Oracle Java, AdoptOpenJDK and all the other distributions, then this is the talk for you!
```

> AdoptOpenJDK is rapidly becoming a leading provider of OpenJDK™ binaries. With over 140 million downloads in the last year, it is now a serious contender for your production usage of Java™. AdoptOpenJDK provides prebuilt OpenJDK™ binaries from a fully open source set of build scripts and infrastructure. This talk will cover how we build on over 15 different platforms, execute over 87 million tests and distribute OpenJDK™ binaries to millions of users. We will also cover how AdoptOpenJDK binaries compare against the Java™ binaries that you use today. If you're curious to understand the difference between OpenJDK™, Oracle Java™, AdoptOpenJDK and all the other distributions, then this is the talk for you!

Reviewer 3: 170 Million

Reviewer 2: ```suggestion
AdoptOpenJDK is rapidly becoming a leading provider of OpenJDK™ binaries. With over 140 million downloads in the last year, it is now a serious contender for your production usage of Java™. AdoptOpenJDK provides prebuilt OpenJDK™ binaries from a fully open source set of build scripts and infrastructure. This talk will cover how we build on over 15 different platforms, execute over 170 million tests and distribute OpenJDK™ binaries to millions of users. We will also cover how AdoptOpenJDK binaries compare against the Java™ binaries that you use today. If you're curious to understand the difference between OpenJDK™, Oracle Java™, AdoptOpenJDK and all the other distributions, then this is the talk for you!
```

> AdoptOpenJDK is rapidly becoming a leading provider of OpenJDK™ binaries. With over 170 million downloads in the last year, it is now a serious contender for your production usage of Java™. AdoptOpenJDK provides prebuilt OpenJDK™ binaries from a fully open source set of build scripts and infrastructure. This talk will cover how we build on over 15 different platforms, execute over 87 million tests and distribute OpenJDK™ binaries to millions of users. We will also cover how AdoptOpenJDK binaries compare against the Java™ binaries that you use today. If you're curious to understand the difference between OpenJDK™, Oracle Java™, AdoptOpenJDK and all the other distributions, then this is the talk for you!

Reviewer 4: One (tm) per term should be sufficient.

Author: updated

## Speakers
**George Adams** is the Java Program Manager at Microsoft and is the recently appointed chair of the Technical Steering Committee at AdoptOpenJDK. Since co-founding AdoptOpenJDK in 2016 he has been leading the community outreach efforts at AdoptOpenJDK and is heavily involved in the Java ecosystem. George also contributes to both the OpenJS Foundation and the Node.js Foundation where he is a core collaborator and plays an active role in several of the workgroups.

> **Hendrik Ebbers** is a co-founder of Karakun AG and lives in Dortmund, Germany. He is the founder and leader of the Java User Group Dortmund and gives talks and presentations in user groups and conferences. His JavaFX book "Mastering JavaFX 8 Controls" was released 2014 by Oracle press. Hendrik is JavaOne Rockstar, JSR expert group member and Java Champion. Hendrik is member of the AdoptOpenJDK TSC.

Reviewer: Typos: 
Should read, "...was released in 2014 by..." and "...is a JavaOne Rockstar..."

Reviewer 2: ```suggestion
**Hendrik Ebbers** is a co-founder of Karakun AG and lives in Dortmund, Germany. He is the founder and leader of the Java User Group Dortmund and gives talks and presentations in user groups and conferences. His JavaFX book "Mastering JavaFX 8 Controls" was released in 2014 by Oracle press. Hendrik is a JavaOne Rockstar, JSR expert group member and Java Champion. Hendrik is member of the AdoptOpenJDK TSC.
```

## Scheduled Events
| Date | JUG | Link |
|---|---|---|
| 21/07/ 2020 18:30 EDT | Knoxville JUG  |  https://www.meetup.com/de-DE/KnoxJava/events/nmfmbrybckbcc/ |
| 06/08/2020  19:00 GMT+3 | JUG Istanbul |  https://www.meetup.com/de-DE/Istanbul-Java-User-Group/events/271767087 |  
| 11/08/2020  11:00 EDT | Connecticut JUG  | tbd |
| 19th-22nd October | Eclipse Con 2020 | https://www.eclipsecon.org/2020/sessions/adoptopenjdk-making-java-free-again |

> ## Upcomming Events

Reviewer: Typo: Upcomming -> Upcoming

Reviewer 2: ```suggestion
## Upcoming Events
```
